python/lsst/ts/lasertracker/t2sa_model.py

```diff
@@ -257,6 +246,7 @@ async def _basic_send_command(self, cmd: str) -> str:
         except asyncio.TimeoutError:
             err_msg = (
                 f"Timed out while waiting for a reply to command {cmd}; disconnecting"
+                f"Read timeout: {self.read_timeout}s. Wait time {time.monotonic()-t0}s."
             )
             self.log.error(err_msg)
             # await self.handle_lost_connection()
```

Comment:
You can use the opportunity and delete commented line, as it is mot likely really not needed.